datastream-common/src/main/idl/com.linkedin.datastream.server.dms.datastream.restspec.json

```diff
@@ -31,6 +31,16 @@
         "type" : "string"
       } ]
     } ],
+    "actions" : [ {
```

Comment:
Can you help me understand the organization of this json? I noticed that actions is inside entities. Sorry, I'm not aware of how this piece of code works.